nox/virtualenv.py

```diff
@@ -62,6 +64,23 @@ def run(self, args, in_venv=True):
             path=self.bin if in_venv else None).run()
 
 
+def locate_via_py(v_maj, v_min):
+    """Find the Python executable using the Windows launcher.
+
+    Code taken from tox (tox/interpreters.py).
+    """
+    ver = "-%s.%s" % (v_maj, v_min)
+    script = "import sys; print(sys.executable)"
+    py_exe = distutils.spawn.find_executable('py')
```

Comment:
TIL about `distutils.spawn.find_executable`! Should we use that over [`py.path.local.sysfind`](http://py.readthedocs.io/en/latest/path.html#py._path.local.LocalPath.sysfind) which we use in [command](https://github.com/jonparrott/nox/blob/f87525654265afdb3da086d4116fdfb5867154f6/nox/command.py#L35)?

Reply:
Honestly, I doubt it matters (I lifted that code from tox unchanged). I'd be inclined to go for consistency and follow current practice by using `py.path.local.sysfind` in both places. I'll change this.

(BTW, there's also `shutil.which` in Python 3.3+. Who said "one obvious way"? :smile:)

Reply:
> (BTW, there's also shutil.which in Python 3.3+. Who said "one obvious way"? 😄)

I'd love to make nox 3+ only. Sigh.

Reply:
I'll also make the change to drop the old approach, as noted in your other comment. I can do the tests too, if you want, but if you have a preferred style/approach for the tests and would rather cover that yourself, that's fine too.

Reply:
We use pytest so as long as it's not terribly different from what's there go for it. :)